cms/djangoapps/contentstore/views/library.py

```diff
@@ -119,6 +119,11 @@ def _create_library(request):
         display_name = request.json['display_name']
         org = request.json['org']
         library = request.json.get('number', None)
+
+        user_perms = get_user_permissions(request.user, None, org=org)
+        if not user_perms & STUDIO_VIEW_USERS:
```

Comment:
this permissions paradigm is really crazy :). Would we also want to check if you have STUDIO_VIEW_CONTENT permissions?